flixel/graphics/tile/FlxDrawTilesItem.hx

```diff
@@ -91,7 +95,11 @@ class FlxDrawTilesItem extends FlxDrawBaseItem<FlxDrawTilesItem>
 			#end
 			
 			tempFlags |= blending;
+			#if (!openfl_legacy && openfl >= "3.3.9")
+			camera.canvas.graphics.drawTiles(graphics.tilesheet, drawData, (camera.antialiasing || antialiasing), tempFlags, shader, position);
```

Comment:
Why do we draw to `camera.canvas.graphics` in this case and `graphics.tilesheet` in the other?